packages/varnish/src/theme/colors.ts

```diff
@@ -75,6 +76,7 @@ export const color2: Dictionary<Color> = {
   N3: new Color('N3', '#AEB7C4', true),
   N4: new Color('N4', '#616C7A', true),
   N5: new Color('N5', '#47515C', true),
+  N6: new Color('N6', '#47515C4D', true), // TODO: validate naming conventions with Jon
```

Comment:
this is n5 with transparency.. mui should be setting this when disabled.. lets find out why it is not

dont add this